src/EventTickets/resources/admin/components/EventFormModal/index.tsx

```diff
@@ -0,0 +1,157 @@
+import {SubmitHandler, useForm} from 'react-hook-form';
+import {__} from '@wordpress/i18n';
+import styles from './EventFormModal.module.scss';
+import FormModal from '../FormModal';
+import EventTicketsApi from '../api';
+
+/**
+ * Get the next sharp hour
+ *
+ * @unreleased
+ */
+const getNextSharpHour = (hoursToAdd: number) => {
+    const date = new Date();
+    date.setHours(date.getHours() + hoursToAdd, 0, 0, 0);
+
+    return date;
+};
+
+/**
+ * Format a given date to be used in datetime inputs
+ *
+ * @unreleased
+ */
+const getDateString = (date: Date) => {
+    if (!date) {
```

Comment:
I think if we are calling `getDateString` and not passing a date then we want an error to occur, so that we know what the problem is. Otherwise, we run the risk of creating downstream issues that are harder to diagnose.

Reply:
Proposal: 21a5f9cb6264aff6e435a5394dfc16915a22020d

Reply:
Is this not already a required parameter? Do we need an extra check here with the typing?

Reply:
What's your suggestion here?

Reply:
Can we just remove the check? Does the compiled typescript not already check this?

Reply:
Oh, finally got what you meant here.

Reply:
d49c81ccfb1338ddae3b183ab8cb83cfe8a832c4